scripts/gen_vimdoc.py

```diff
@@ -38,37 +39,91 @@
   - Parameters are omitted for the `void` and `Error *` types, or if the
     parameter is marked as [out].
   - Each function documentation is separated by a single line.
+
+TODO:
+- Handle defs like 'vim.g' and others getting generated
+
+- Handle local functions being exported, even though they should not be
+    - I think I fixed this, but there is a new problem
```

Comment:
I don't really see this as worth spending time on. Prefix the functions with underscore, problem solved.